src/lib/services/project-service.ts

```diff
@@ -390,6 +390,14 @@ export default class ProjectService {
             );
         }
 
+        const archivedToggles = await this.featureToggleStore.getAll({
+            project: id,
+            archived: true,
+        });
+
+        await this.featureToggleStore.batchDelete(
```

Comment:
We should also post events for deletions

Reply:
If we're deleting the project, where will these go?

Reply:
Global event list.

Reply:
Update - we have a global event log. That's where this will go

Reply:
Good catch, I'll fix, thanks!